src/components/Profile/Profile.jsx

```diff
@@ -9,7 +9,7 @@ const Profile = (props) => {
   return (
     <div>
       <ProfileInfo />
-      <MyPosts posts={props.posts} />
+      <MyPosts posts={props.posts} addPost={props.addPost} newPostText={props.posts.newPostText} updateNewPostText={props.updateNewPostText}/>
```

Comment:
Whenever you've got more than 3 props, consider formatting you JSX like this:
![image](https://user-images.githubusercontent.com/25229823/77856219-56cb0900-71fe-11ea-8696-2c165481a395.png)

Reply:
done.